nucliadb/nucliadb/search/api/v1/resource.py

```diff
@@ -86,8 +86,6 @@ async def search(
     range_modification_end: Optional[datetime] = None,
     features: List[SearchOptions] = [
         SearchOptions.PARAGRAPH,
-        SearchOptions.VECTOR,
```

Comment:
Like this, if the user requests vectors or relations we will simply ignore it.
Have you considered returning a 4xx instead?

Reply:
I've just removed the option, as we don't want to provide it